timescaledb/tutorials/grafana/visualizations/pie-chart.md

```diff
@@ -0,0 +1,167 @@
+
+Pie charts can answer questions like:
+
+*   Which was the least traded stock volume last month?
+*   Which stock has the highest traded stock today?
+*   Who had the highest percentage of accrued votes in the last election?
+
+This tutorial shows you how to:
+
+*   [Create a pie chart](#create-a-pie-chart-with-preaggregated-data)
+  with pre-aggregated data using `time_bucket()`.
+*   [Create a donut chart](#create-a-donut-chart-with-volume-transactions) to show volume of stock transactions.
+
+A pie chart can be in the traditional pie style or in donut style. Both display the
+same information. This tutorial shows you how to create either.
+
+## Prerequisites
+
+<GrafanaVizPrereqs />
+
+## Create a pie chart with preaggregated data
+
+Create a pie chart visualization using the data in the table `stocks_real_time`.
+
+<procedure>
+
+### Creating a pie chat with preaggregated data
+
+1.  In the query editor, use this SQL to query a pie chart dataset. Use the
+    variable `$bucket interval` for the time period covered by the pie chart:
+
+    ```sql
+    SELECT
+        time_bucket('$bucket_interval', time) AS time,
+        symbol,
+        AVG(price) AS price
+    FROM stocks_real_time srt
+    WHERE symbol IN ($symbol)
+        AND time >= $__timeFrom()::timestamptz AND time < $__timeTo()::timestamptz
+    GROUP BY time_bucket('$bucket_interval', time), symbol
+    ORDER BY time_bucket('$bucket_interval', time), symbol;
+    ```
+
+1.  Fetch all company symbols from the dataset in the
+   [Getting Started Tutorial][gsg-data] with this query:
+
+   ```sql
+    SELECT
+        DISTINCT symbol FROM company ORDER BY symbol ASC;
+    ```
+
+1.  In the Grafana dashboard, navigate to the settings page for your panel. In
+   the `Variables` section:
+
+    *   In the `Name` field, give your symbol a name.
+    *   In the `Type` field, select `Query`.
+    *   In the `Selection options` field, enable `multi-value`.
+
+1.  In the Grafana dashboard, in the `Dashboard variable` field, select the stocks
+    to graph. Adjust the time range of the dashboard if needed. Make sure the
+    returned data has a column named `time` that contains timestamps. The
+    timestamps should be in ascending order. Otherwise, you get an error. The
+    returned data looks like this:
+
+    <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tutorials/visualizations/piechart/tabledata1.png" alt="Screenshot of the table view of valid time-series data for stocks."/>
+
+1.  In the `Visualizations` field, select `Pie chart`. Grafana turns the query
+    into a pie chart. This example shows a pie chart price distribution of JPM, IBM, AAPL, AMD, and CVS stocks which has 22%, 23%, 24%, 15%, and 16%, respectively, within a specific period. The returned data had lots of data, but only the lowest values for each stock were displayed because it was the selected option in the calculation field.
```

Comment:
```suggestion
1.  In the `Visualizations` field, select `Pie chart`. Grafana turns the query
    into a pie chart. This example shows a pie chart price distribution of 
    JPM, IBM, AAPL, AMD, and CVS stocks which has 22%, 23%, 24%, 
    15%, and 16%, respectively, within a specific period. The returned data 
    has lots of information, but only the lowest values for each stock are 
    displayed, because of the options selected in the calculation field.
```